src/neuroconv/nwbconverter.py

```diff
@@ -158,10 +159,24 @@ def create_nwbfile(self, metadata: Optional[dict] = None, conversion_options: Op
 
     def add_to_nwbfile(self, nwbfile: NWBFile, metadata, conversion_options: Optional[dict] = None) -> None:
         conversion_options = conversion_options or dict()
-        for interface_name, data_interface in self.data_interface_objects.items():
-            data_interface.add_to_nwbfile(
-                nwbfile=nwbfile, metadata=metadata, **conversion_options.get(interface_name, dict())
-            )
+        for interface_key, data_interface in self.data_interface_objects.items():
+            if isinstance(data_interface, NWBConverter):
+                subconverter_kwargs = dict(nwbfile=nwbfile, metadata=metadata)
+
+                # Certain subconverters fully expose control over their interfaces conversion options
```

Comment:
Which interfaces?

Reply:
Bruker/Miniscope to name a few. Haven't compiled a complete list

Reply:
Ah, but then we lose any differences that would have resulted from special `get_metadata` or `add_to_nwbfile` (including conversion option exposure) overrides in the subcoverters

Reply:
I will check the examples you mentioned to think better about this.